### PR TITLE
DSF-102 Contract for response updates

### PR DESCRIPTION
**Pull request by ChristianAMartin**
DSF-102 Addition of invalid reference unit test
Change to request reference pattern added
Fixed properties for response destinations

### CODE REVIEW

**Review by forjayssake — APPROVED** [review]

**Review by terado — DISMISSED** [review]

**Review by terado — APPROVED** [review]

**Review by forjayssake — APPROVED** [review]

File: application/roaddistance/openapi_schemas/json/dos_road_distance_api_response.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "example": {
+    "status": 200,
+    "message": "",
+    "transactionid": "43c31af7-1f53-470f-9edc-fed8f447dc8f",
+    "destinations": {
+      "123409584": 12546,
+      "567834756": 4563
+    },
+    "unreachable": [
+      674756978,
```
Comment (terado) [inline application/roaddistance/openapi_schemas/json/dos_road_distance_api_response.json:52]: These should be strings

File: application/roaddistance/openapi_schemas/json/travel_time_api.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "type": "array",
       "description": "List of point locations",
       "minItems": 1,
-      "maxItems": 2000,
```
Comment (terado) [inline application/roaddistance/openapi_schemas/json/travel_time_api.json:9]: Revert this as this is not the protobuf one that has 3k, this only has 2k

File: application/roaddistance/openapi_schemas/yaml/dos_road_distance_api.yaml [hunk 3]
```diff
@@ -88,3 +88,99 @@ paths:
...
+                      description: destination reference
+                example:
+                  status: 200
+                  message: ""
+                  transactionid: "43c31af7-1f53-470f-9edc-fed8f447dc8f"
+                  destinations:
+                    123409584: 12546
+                    567834756: 4563
+                  unreachable:
+                    - 674756978
```
Comment (terado) [inline application/roaddistance/openapi_schemas/yaml/dos_road_distance_api.yaml:134]: These will need to be strings

File: application/roaddistance/openapi_schemas/yaml/dos_road_distance_api.yaml [hunk 4]
```diff
@@ -88,3 +88,99 @@ paths:
...
+                    type: integer
+                    description: HTTP status code
+                  message:
+                    type: string
+                    description: Specifies an internal server error occurred
+                example:
+                  status: 500
+                  message: "An internal server error occurred"
+
+components:
```
Comment (terado) [inline application/roaddistance/openapi_schemas/yaml/dos_road_distance_api.yaml:180]: This needs to go

File: application/roaddistance/openapi_schemas/yaml/travel_time_api.yaml [hunk 5]
```diff
@@ -22,7 +22,7 @@ paths:
                   type: array
                   description: List of point locations
                   minItems: 1
-                  maxItems: 2000
```
Comment (terado) [inline application/roaddistance/openapi_schemas/yaml/travel_time_api.yaml:25]: Revert these to 2k as above

**Review by forjayssake — APPROVED** [review]

**Review by terado — APPROVED** [review]

**Review by livseywilliams — APPROVED** [review]

**Pull request merged**